mne_bids/utils.py

```diff
@@ -518,6 +519,24 @@ def _import_nibabel(why="work with MRI data"):
         return nibabel
 
 
+# better example sorting, without relying on numbers in example titles
+with open(Path(__file__).parents[1] / "doc" / "example_order.json") as fid:
+    EXAMPLE_ORDER = json.load(fid)
```

Comment:
NB: the JSON file won't be included in the built wheel, but that shouldn't matter as it's only needed when building our docs, so we can assume a clone of the repo in that case.